srcCpp/perftest_publisher.cxx

```diff
@@ -88,6 +89,16 @@ int subscriber_main()
 
 int perftest_cpp::Run(int argc, char *argv[])
 {
+    DDS_ProductVersion_t ddsV = perftest_cpp::GetDDSVersion();
+    Perftest_ProductVersion_t perftestV = perftest_cpp::GetPerftestVersion();
+    printf("RTI DDS version: %d.%d.%d\n",
+            ddsV.major,
+            ddsV.minor,
+            ddsV.release);
+    printf("RTI Perftest version: %d.%d.%d \n",
+            perftestV.major,
+            perftestV.minor,
+            perftestV.release);
```

Comment:
All this can be encapsulated into a function: PrintVersion().

Also, can we have a single printf with this?

```
RTI Perftest 2.3.2 (RTI Connext DDS 5.3.1)
```